docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -87,7 +86,8 @@ subquestion: |
 
   ### What you need
 
-  You need to know about your old and open cases. That includes
+  You need to know about your closed cases and open cases.
+  That includes
```

Comment:
Nit. The line break here is odd. Seems like it could be combined with the next line.

Reply:
Eh, might as well fix it 👍